pkg/operator/config/prometheus/monitor.yaml

```diff
@@ -17,3 +17,22 @@ spec:
   selector:
     matchLabels:
       control-plane: controller-manager
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    name: nodeagent
+  name: nodeagent-metrics-monitor
+  namespace: thundernetes-system
```

Comment:
namespace should be just "system". Kustomize adds the "thundernetes-" prefix when building the YAML files.